aptos-move/block-executor/src/view.rs

```diff
@@ -734,7 +739,11 @@ impl<'a, T: Transaction, X: Executable> ResourceGroupState<T> for ParallelState<
                 },
                 Err(Dependency(dep_idx)) => {
                     if !wait_for_dependency(self.scheduler, txn_idx, dep_idx) {
-                        bail!("Interrupted as block execution was halted");
+                        // TODO[agg_v2]: halting should be handled like ReadResult.
```

Comment:
you mean add new variant to GroupReadResult ? why is SPECULATIVE_EXECUTION_ABORT_ERROR not enough?

Reply:
Right now we have `PartialVMResult<GroupReadResult>` which is a bit awkward?